<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d1_Screen_6</class>
 <widget class="QWidget" name="d1_Screen_6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="d1_6_widgetBackground" native="true">
     <property name="styleSheet">
      <string notr="true">#d1_6_widgetBackground{
	border-image: url(:/d1_whiteFonYellow - Copy.png);
	background-color: rgb(255, 255, 255);
	border-radius:  30px;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="d1_6_vlBackground">
      <item>
       <widget class="QWidget" name="d1_6_widgetTop" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="d1_6_lblInfo">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>81</red>
                 <green>17</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>81</red>
                 <green>17</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="96">
                 <red>164</red>
                 <green>166</green>
                 <blue>168</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>Baloo Cyrillic</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Оплата  QR-Code</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="d1_6_btnClose">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#d1_6_btnClose{
	border-image: url(:/d1_btnClose.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="d1_6_widgetContent" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="d1_6_vlContent">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="d1_6_lblInfoQrCodeTop">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>81</red>
                 <green>17</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>81</red>
                 <green>17</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>190</red>
                 <green>190</green>
                 <blue>190</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>Baloo Cyrillic</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>QR-Code</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="d1_6_lblQrCode">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="d1_resource/d1_resource.qrc">:/qiwiQr.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="d1_6_lblInfoQrCodeBottom">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>81</red>
                 <green>17</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>81</red>
                 <green>17</green>
                 <blue>68</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="96">
                 <red>164</red>
                 <green>166</green>
                 <blue>168</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>Baloo Cyrillic</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Внимание !</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="d1_6_widgetBtnBuy" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="d1_6_btnBack">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">#d1_6_btnBack{
	border-image: url(:/d1_btnBack.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>454</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="d1_resource/d1_resource.qrc"/>
 </resources>
 <connections/>
</ui>
